mlcluster-vpc.template

```diff
@@ -101,7 +107,19 @@ Parameters:
     AllowedValues:
       - standard
       - gp2
+      - gp3
     Default: gp2
+  VolumeIOPS:
+    Description: The number of I/O operations per second for a gp3 volume
```

Comment:
Do we need to add more description similar to Throughput ? "This parameter is only applicable for a gp3 volume type" ?